pandas/tests/frame/test_combine_concat.py

```diff
@@ -347,6 +347,20 @@ def test_concat_named_keys(self):
                                              names=[None, None]))
         assert_frame_equal(concatted_unnamed, expected_unnamed)
 
+    def test_concat_axis_parameter(self):
+        # GH 14369
+        df1 = pd.DataFrame({'A': [0.1, 0.2]}, index=range(2))
+        df2 = pd.DataFrame({'A': [0.3, 0.4]}, index=range(2))
+        expected_row = pd.DataFrame(
+            {'A': [0.1, 0.2, 0.3, 0.4]}, index=[0, 1, 0, 1])
+        concatted_row = pd.concat([df1, df2], axis='rows')
```

Comment:
Probably already tested elsewhere, but can you add for completeness in this test also the `index=0` case and check output of that as well? (and the same for axis=1 below)